packages/notebook/src/widget.ts

```diff
@@ -925,34 +930,6 @@ export class StaticNotebook extends WindowedList {
     await cell.ready;
   }
 
-  private _renderCSSAndJSOutputs(
```

Comment:
No because outputs are not remove from the DOM as they may have internal state or side effects. This is the reason that method existed to render but the opposite does not.

This is one of the notebook windowing trickiest part:

https://github.com/jupyterlab/jupyterlab/blob/80dd3d65e3cde89d6b7f5dccd7fdf38f18b453a1/packages/cells/src/widget.ts#L968

> Obviously this PR will induce larger DOM tree for windowing notebook as it will match the one seen by an user after fully scrolling it.